site/default/static/map.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
I think every .js file is fine from within `node_modules/can-*` is fine. You can exclude `_test` files if you want, but I don't think it will hurt anything to have "too much" configuration.